STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/utils/ApiEnvironment.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Infomaniak SwissTransfer - Multiplatform
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.multiplatform_swisstransfer.common.utils
+
+sealed class ApiEnvironment(val baseUrl: String) {
+    data object Preprod : ApiEnvironment("https://swisstransfer.preprod.dev.infomaniak.ch")
+    data object Prod : ApiEnvironment("https://www.swisstransfer.com")
```

Comment:
We probably don't need www.
```suggestion
    data object Prod : ApiEnvironment("https://swisstransfer.com")
```